microcosm_flask/converters.py

```diff
@@ -11,3 +11,17 @@ def configure_uuid(graph):
 
     """
     return FlaskUUID(graph.flask)
+
+
+def str_to_bool(value):
+    """
+    Convert a string value to boolean.
+    Similar to distutils.util.strtobool but returns a boolean instead of an int.
```

Comment:
I'm a little worried about the `similar to .. but returns a boolean instead of an int` - mostly because `microcosm-flask` is used everywhere and this could have some side-effects that will be tricky to debug / will only show up in a while from now when someone pulls in latest microcosm-flask.

So if you want to go with this I would test thoroughly with at least 1-2 services that it works as expected